src/AuthContext.tsx

```diff
@@ -117,10 +115,20 @@ export const AuthProvider = ({ authConfig, children }: IAuthProvider) => {
 
   function handleExpiredRefreshToken(initial = false): void {
     // If it's the first page load, OR there is no sessionExpire callback, we trigger a new login
-    if (initial) return login()
+    if (initial) {
+      login()
+      return
+    }
```

Comment:
Ah, I too would have preferred `return login()`, but this is equally as understandable.